Makefile

```diff
@@ -9,13 +9,22 @@ test: ## Run tests
 	go test ./... \
 		-race \
 		--ginkgo.focus="${FOCUS}" \
+		--ginkgo.skip="benchmark" \
```

Comment:
This is going to grow with other types of skips. Is there another way we could do this that doesn't make the target explicitly aware?

Reply:
That's a good point. I can't think of any new CI makefile targets that we'd include, but one way is to label all the non benchmarked ones with "CI" and all benchmark tests as "benchmark". This way any new ones would need to just add this label themselves in the future. Thoughts?